far/editor.cpp

```diff
@@ -319,14 +324,26 @@ void Editor::ShowEditor()
 	const Edit::ShowInfo info{ LeftPos, CurPos };
 	auto Y = m_Where.top;
 
+	const auto LineNumColor = LineNumWidth > 0 ? colors::PaletteColorToFarColor(COL_EDITORLINENUMBERS) : FarColor{};
```

Comment:
No need to have a condition here, `PaletteColorToFarColor` is cheap.
Just `const auto& LineNumColor = colors::PaletteColorToFarColor(COL_EDITORLINENUMBERS);` will do.